compute/admin_guide/vulnerability_management/agentless_scanning.adoc

```diff
@@ -2,99 +2,38 @@
 
 Agentless scanning lets you inspect the risks and vulnerabilities of a virtual machine without having to install an agent or affecting the execution of the instance.
 Prisma Cloud gives you the flexibility to choose between agentless and agent-based security using Defenders.
-Currently, Prisma Cloud supports agentless scanning on AWS hosts for vulnerabilities.
+Currently, Prisma Cloud supports agentless scanning on AWS, GCP and Azure hosts for vulnerabilities and compliance.
 Agentless capabilities will continue to be enhanced over future releases in parallel with Defender capabilities.
```

Comment:
Can we delete this forward looking statement? All our features are enhanced when we can.

Reply:
I thought this was to reflect the capability added with the update. @Aqsaxvv092 can you confirm whether or not this can stay?

Reply:
it is true, We support all three providers. You can see this in the product.

Reply:
People have mistaken agentless as a one time enhancement similar to other enhancement features we do randomly with no proper roadmap. Agentless is new, we must make sure there is trust that it has its dedicated roadmap and not a one time feature.

Reply:
My question was not about the sentence "Currently, Prisma Cloud supports agentless scanning on AWS, GCP and Azure hosts for vulnerabilities and compliance."

It was about "Agentless capabilities will continue to be enhanced over future releases in parallel with Defender capabilities."